Contributing.md

```diff
@@ -46,10 +40,25 @@ Before diving in, follow the [Manual Installation Instructions](docs/Manual%20In
 * Add new features:
   * If you would like to become involved in the development effort, please hop
    on our [OFTC channel at #activecm](https://webchat.oftc.net/?channels=activecm)
-   and chat about what is currently being worked on.
+   and chat about what is currently being worked on
 
-All of these tasks ultimately culminate in a pull request being issued,
-reviewed, and merged. 
+### Running Static Tests
+* Golint
+  * Install [golint](https://github.com/golang/lint)
+  * Run `golint ./...` from the root RITA directory
+  * Fix any errors and run golint again to verify
+* Gofmt
+  * Run `gofmt -l .` from the root RITA directory to identify files containing styling errors
```

Comment:
Same here `gofmt -l . | grep -v '^vendor/'`. I'm not sure what to do about the one that makes changes.  Hopefully it doesn't matter there.

Reply:
I don't believe it would matter since it won't break anything with just styling changes, and the vendor directory is ignored by git. Running it just now after setting up vendored dependencies and reinstalling RITA worked fine.